quickstarts/releaseiq/config.yml

```diff
@@ -24,7 +24,7 @@ summary: |
   Monitor ReleaseIQ Performance with NewRelic's ReleaseIQ Agent.
 
 # Support level: New Relic | Verified | Community (required)
-level: Community
+level: Verified
```

Comment:
Thanks for this change I wanted to confirm this complies with our definition of verified before changing this level.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#support-levels

Verified

    Verified for quality by New Relic
    Created by New Relics employees or partners
    Supported by individual authors or partners

Reply:
yes, it's created by ReleaseIQ, which is a partner that we're working with

Reply:
Thank you!